scripts/setup_gce_vm_instance.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+echo ' '
+echo 'This script assists you in creating a Google Cloud Compute Engine VM instance.'
+echo 'Please note that currently this script convenience during DEVELOPMENT by automating'
+echo 'the default options for most GCE VMs and does not include all available options, '
+echo 'such as customizing disk size, type, etc. We could add more features in the future.'
+echo ' '
+echo 'The script assumes you have:'
+echo ' - installed the Google Cloud SDK: https://cloud.google.com/sdk/downloads'
+echo ' - subscribed your @gmail.com or other google account to M-Lab Discuss: '
+echo '   https://groups.google.com/a/measurementlab.net/forum/#!forum/discuss'
+echo ' - created a project for your Piecewise install in the Google Cloud Console: '
+echo '   https://console.cloud.google.com/'
+echo ' - configured your installed Google Cloud SDK with the Google account and '
+echo '   project name above'
+echo ' '
+echo 'If you need to cancel, press CTRL-C, complete the requirements above and '
+echo 'then re-run this script.'
+echo ' '
+read -p 'Name of your GCE VM instance: ' vmName
+read -p 'Image family (debian-8 or rhel-7): ' imageFam
```

Comment:
Rather than reading in these values from direct user input, you might consider letting the user pass them in as argument. Once done, it's easy to assign them to internal variables with better names, and even to include defaults. For example:

```
if [[ -n "$1 ]]; then
  vmName=$1
else
  echo "You must pass at least a VM name."
  exit 1
fi

imageFam=${1-debian-8}
```

Then the script could be called with something like:

`setup_gce_vm_instance.sh myvirtualmachine`

or for RedHat:

`setup_gce_vm_instance.sh myvirtualmachine rhel-7`